src/java/htsjdk/samtools/CRAMFileReader.java

```diff
@@ -99,36 +92,43 @@ public CRAMFileReader(final File cramFile, final InputStream inputStream,
      *                        reference sequences
      */
     public CRAMFileReader(final File cramFile, final File indexFile,
-                          final ReferenceSource referenceSource) {
-        if (cramFile == null)
-            throw new IllegalArgumentException("File is required.");
-
-        this.cramFile = cramFile;
-        this.mIndexFile = indexFile;
-        this.referenceSource = referenceSource;
+                          final ReferenceSource referenceSource) throws FileNotFoundException,IOException {
```

Comment:
and here: it's bad style to have 2 exceptions declared and one of them be a subclass of the other: FileNotFoundException extends IOException